packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -288,6 +295,24 @@ export class SingleAddressWallet implements ObservableWallet {
       tip$: this.tip$,
       transactionsHistoryStore: stores.transactions
     });
+
+    this.#rollbackSubscription.add(
```

Comment:
I think this would be a little clearer
```suggestion
    this.#rollbackSubscription =  createTransactionReemitter(...).subscribe(async (rollbackTransaction) => {})
```

Reply:
> I think this would be a little clearer

The advantage of `add` (or at least the way I've used it in the past), is that you don't have to check if the subscription exists and subscribed to, in order to unsubscribe it.
Offtopic: it's also a great way to bundle multiple subscriptions that you want to cleanup at a later time.

Reply:
I see, but in this case subscription is being set in constructor so it will always exist.

> Offtopic: it's also a great way to bundle multiple subscriptions that you want to cleanup at a later time.

Yep that's a nice way to bundle subscriptions

Reply:
Done in: [ddb6af8](https://github.com/input-output-hk/cardano-js-sdk/pull/351/commits/ddb6af800a591149f87a2668b4da04d0470a29c7)